src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 410px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+    &__image-container {
+      margin: 32px 19px 0;
+    }
+
+    &__image {
+      width: 160px;
+      height: 134px;
+      flex-shrink: 0;
+    }
+
+    &__name {
+      margin: 40px 16px 4px;
+      color: $color-main;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    &__code {
+      color: $color-secondary;
+      margin-left: 16px;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+
+    &__price {
+      display: flex;
+      justify-content: space-between;
+    }
+
+    &__price-word {
+      color: $color-secondary;
+      margin-top: 24px;
+      margin-left: 16px;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &__price-number {
+      color: $color-main;
+      text-align: right;
+      margin-top: 24px;
+      margin-right: 16px;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+
+    &__buy {
+      padding: 0;
+      margin: 16px;
+      width: 166px;
+      height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Use padding instead

Reply:
not fixed